common/src/query.rs

```diff
@@ -0,0 +1,43 @@
+use crate::tx;
+use getset::Getters;
+use serde::{Deserialize, Serialize};
+
+#[derive(Debug, Clone, Serialize, Deserialize, Getters)]
+#[getset(get = "pub")]
+pub struct GetResultsQuery {
+    request_tx_hash: tx::TxHash,
+    start: u32,
+    size: u32,
+}
+
+impl GetResultsQuery {
+    pub fn new(request_tx_hash: tx::TxHash, start: u32, size: u32) -> Self {
+        Self { request_tx_hash, start, size }
+    }
+}
+
+#[derive(Debug, Clone, Serialize, Deserialize, Getters)]
+#[getset(get = "pub")]
+pub struct GetTrustUpdateQuery {
+    from: tx::TxHash,
```

Comment:
`from` should be optional too

Reply:
Done https://github.com/openrankprotocol/openrank/pull/171/commits/2bf5128a1db45c4470e86886b3f744897b080f00